include/psp2/npdrmpackage.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * \file
+ * \brief Header file which defines NpDrmPackage related variables and functions
+ *
+ * Copyright (C) 2015 PSP2SDK Project
```

Comment:
Is this date thing really necessary? It seems to be an unnecessary burden, although minor.